app/controllers/chores_controller.rb

```diff
@@ -17,6 +19,15 @@ def create
     redirect_to @chore
   end
 
+  def update
```

Comment:
checkout this example of update and compare it to yours https://www.railstutorial.org/book/updating_and_deleting_users#sec-unsuccessful_edits